pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     "BayesicFitting>=3.0.1",
     "crds>=11.17.14",
     "drizzle>=1.15.0",
-    "gwcs>=0.21.0,<0.22.0",
+    "gwcs>=0.22.0,<0.23.0",
```

Comment:
I think the latest that's out is 0.21.0 and I wnted to skip one version in case we need it for non-breaking changes. Should this be >=0.21, <0.23.0

Reply:
```suggestion
    "gwcs>=0.21.0,<0.23.0",
```

Reply:
Yes, the latest gwcs that's in the wild right now is 0.21.0. So this allows for a non-breaking 0.22.0, if it's needed.